ambry-clustermap/src/main/java/com/github/ambry/clustermap/HelixClusterManager.java

```diff
@@ -461,10 +486,14 @@ public void registerClusterMapListener(ClusterMapChangeListener clusterMapChange
    */
   @Override
   public void close() {
-    for (DcInfo dcInfo : dcToDcInfo.values()) {
-      dcInfo.close();
+    if (clusterMapConfig.clusterMapUseAggregatedView) {
+      helixAggregatedViewClusterInfo.close();
+    } else {
+      for (DcInfo helixDcInfo : dcToDcInfo.values()) {
+        helixDcInfo.close();
```

Comment:
Even clusterMapUseAggregatedView is true, dcToDCInfo map is still populated.  Do you still need to loop through DCInfo and call close?